Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -52,6 +54,17 @@ public class RequestTrackingTelemetryModule : ITelemetryModule
         /// </summary>
         public bool EnableAccessControlExposeHeader { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether requestTelemetry.Url and requestTelemetry.Source are disabled.
```

Comment:
So cookies for user id and session id are cheap to collect?

Reply:
At least it is not in current testing methodology we use. We can take a look into measuring this as a next step.